odoo/addons/base/ir/ir_http.py

```diff
@@ -161,7 +189,10 @@ def _handle_exception(cls, exception):
             attach = cls._serve_attachment()
             if attach:
                 return attach
-
+            website_page = cls._serve_page()
+            if website_page:
```

Comment:
In all case, you have a return 

So you will always enter here and ignore following code